README_BUILD.md

```diff
@@ -49,11 +49,13 @@ When importing or creating custom types, the following considerations will allow
 build your types.
 
 * See the notes in Building above.
-* The builder will import your custom types into the vendored resources. They are required to be there for the module to successfully find them during a Puppet catalog application.
+* Do not try to include your module in the import folder by default. When building the first time or during a clean, the builder will delete all files and folders in this directory. It is preferred that you keep those custom modules in a separate location. A suggested location is `build/vendor/custom` within this repository.
```

Comment:
Moved to the top. However I'm not sure about prescribing a location within the repository for storing. Mostly due to updating a fork from source control and limitations on being able to give back fixes through Pull Requests (trying not to accidentally disclose possibly sensitive information).